pkg/ddc/alluxio/transform.go

```diff
@@ -81,6 +82,20 @@ func (e *AlluxioEngine) transform(runtime *datav1alpha1.AlluxioRuntime) (value *
 	// 8.set optimization parameters
 	e.optimizeDefaultProperties(runtime, value)
 
+	var isHTTP = true
```

Comment:
Could you wrapper this logic into a single function? And add a test function for it?